src/components/sidebar/Sidebar/index.tsx

```diff
@@ -58,7 +58,13 @@ const Sidebar = (): ReactElement => {
 
         <IndexingStatus />
       </div>
-      <Drawer variant="temporary" anchor="left" open={isDrawerOpen} onClose={onDrawerToggle}>
+      <Drawer
+        variant="temporary"
+        anchor="left"
+        open={isDrawerOpen}
+        onClose={onDrawerToggle}
+        ModalProps={{ keepMounted: true }}
```

Comment:
Will it remain mounted after it was rendered once, or will it be mounted from the start? If it's the latter, I would instead look into removing the animation delay like I did for the BarchSidebar.

Reply:
The animation doesn't cause the delay but the computations we do inside the nested components. The Sidebar unmounts every time the user closes it so it keeps doing those computations every time it opens again. We should definitely look into improving performance on those computations but this is out of scope for this fix.

Reply:
To avoid making extra requests on page load, I would suggest doing something like this then:

```js
const [keepMounted, setKeepMounted] = useState(false)

useEffect() => setKeepMounted(true), [])
```